cpg-core/src/main/java/de/fraunhofer/aisec/cpg/graph/Extensions.kt

```diff
@@ -71,9 +71,9 @@ inline fun <reified T : Node> Node.dfgFrom(): List<T> {
 /** This function returns the *first* node that matches the name on the supplied list of nodes. */
 fun <T : Node> Collection<T>?.byNameOrNull(lookup: String, modifier: SearchModifier): T? {
     return if (modifier == SearchModifier.NONE) {
-        this?.firstOrNull { it.name == lookup }
+        this?.firstOrNull { it.name.endsWith(lookup) }
```

Comment:
And here too, but maybe I just overlook why this is not an issue. I would have thought that one still has to split by delimiter and compare the last element in the list